Data/Csv/Conversion.hs

```diff
@@ -715,6 +716,18 @@ instance ToField Char where
     toField = toField . T.encodeUtf8 . T.singleton
     {-# INLINE toField #-}
 
+-- | Accepts the same syntax as 'rational'. Ignores whitespace.
+instance FromField Scientific where
+  parseField s = case parseOnly (ws *> A8.scientific <* ws) s of
+                   Left err -> typeError "Scientific" s (Just err)
+                   Right n  -> pure n
+  {-# INLINE parseField #-}
```

Comment:
note to myself: at some point I should figure out whether those INLINEs sprinkled throughout cassava are really a good idea, and whether we shouldn't rather use INLINEABLE...